### PR TITLE
feat: hometask1

### PR DESCRIPTION
**Pull request by leandrocunha**
To test the homework's tasks, you can run:
* `npm run task1.1`
* `npm run task1.2`
* `npm run task1.3:1`
* `npm run task1.3:1:build`
* `npm run task1.3:2`
* `npm run task1.3:2:build`

### CODE REVIEW

**Comment by WojciechSikora** [comment]
I think that task1.1 can be changed that will not require stoping to batch to rerun it.
I think that's what was meant in instruction's point:

> The program should be running in a stand-by mode and should not be terminated after the first-stringprocessing.

File: src/task1/1.1/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+
+/**
+ * Initialize the listener to receive the answer
+ */
+process.stdin.on("data", data => {
+    const output = revertStr(data);
+
+    process.stdout.write("> " + output + "\n");
+    return process.exit();
```
Comment (WojciechSikora) [inline src/task1/1.1/index.js]: if you remove return statement you don;t have to terminate the program to get another string reversed

File: src/task1/1.2/index.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+
+/**
+ * Call csv() package and start the read/write
+ */
+csv(csvOptions)
+    .fromFile(srcFile)
+    .then(data => { 
+        writeJson(newFile, data);
+    });
```
Comment (WojciechSikora) [inline src/task1/1.2/index.js]: will you change the code or add new .js file using stream.pipepline() according to  requirement from hometask pdf:

> Task 1.2is fulfilled to the full extent;the file is not loaded fully in the RAM(pipeline method https://nodejs.org/api/stream.html#stream_stream_pipeline_streams_callback).

**Comment by leandrocunha** [comment]
@WojciechSikora you could review the enhancements that I did base on your suggestions.

**Comment by WojciechSikora** [comment]
from my POV you can merge this PR

**Pull request merged**